dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -297,14 +301,26 @@ private void configureProxyAuthentication() {
         }
     }
 
-    private void configureAuthentication(final MavenRepository repository) {
+    private void configureAuthentication(final MavenRepository repository) throws InstallException {
 
         if (repository.getUserId() != null && repository.getPassword() != null &&
             envMap.get("https.proxyUser") == null && envMap.get("http.proxyUser") == null) {
+            String decodedPwd = repository.getPassword();
+            try {
+                decodedPwd = PasswordUtil.decode(repository.getPassword());
+            } catch (InvalidPasswordDecodingException ipde) {
+                decodedPwd = repository.getPassword();
+                logger.log(Level.FINE,
+                           Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_TOOL_PWD_NOT_ENCRYPTED")
+                                       + InstallUtils.NEWLINE);
+            } catch (UnsupportedCryptoAlgorithmException ucae) {
+                throw new InstallException(Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_TOOL_PWD_CRYPTO_UNSUPPORTED"));
+            }
+            final String pwd = decodedPwd;
```

Comment:
I'm not a fan of saving passwords in plain text variables. That might be a standard higher than we adhere to on open liberty but it's generally a safe practice to be scared of doing such a thing.